app/controllers/application_controller.rb

```diff
@@ -1,2 +1,2 @@
-class ApplicationController < ActionController
+class ApplicationController < ActionController::Base #bug 1 ::base not written
```

Comment:
you do not need to add comment for each bug but well done fishing all the bugs 👏

Reply:
Thank you.